whats-new-panel.yaml

```diff
@@ -104,3 +62,41 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_SUPERCOOKIE_85
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_SUPERCOOKIE_85
+    published_date: 1614038400000
```

Comment:
This converts to "Feb 23 2021", shall we set it as "Jan 26 2021" instead?

Reply:
Thanks for catching!